### PR TITLE
Update path to vulkan_samples.app in MacOS

### PR DESCRIPTION
**Pull request by chmosquera**
## Description
Adjusted the instructions for building to MacOS by correcting the folder name where the application is built to.

Since the change is a minor instructional update, I did not do any code or build checks. However, I did verify that the command does what's expected: call the application `--help` command.

## General Checklist:

Please ensure the following points are checked:

- [x] My code follows the [coding style](https://github.com/KhronosGroup/Vulkan-Samples/tree/main/CONTRIBUTING.adoc#Code-Style)
- [x] I have reviewed file [licenses](https://github.com/KhronosGroup/Vulkan-Samples/tree/main/CONTRIBUTING.adoc#Copyright-Notice-and-License-Template)
- [x] I have commented any added functions (in line with Doxygen)
- [x] I have commented any code that could be hard to understand
- [x] My changes do not add any new compiler warnings
- [x] My changes do not add any new validation layer errors or warnings
- [x] I have used existing framework/helper functions where possible
- [x] My changes do not add any regressions
- [x] I have tested every sample to ensure everything runs correctly
- [x] This PR describes the scope and expected impact of the changes I am making

 Note: The Samples CI runs a number of checks including:
 - [ ] I have updated the header Copyright to reflect the current year (CI build will fail if Copyright is out of date)
 - [ ] My changes build on Windows, Linux, macOS and Android. Otherwise I have [documented any exceptions](https://github.com/KhronosGroup/Vulkan-Samples/tree/main/CONTRIBUTING.adoc#General-Requirements)

 If this PR contains framework changes:
 - [ ] I did a full batch run using the `batch` command line argument to make sure all samples still work properly

## Sample Checklist

If your PR contains a new or modified sample, these further checks must be carried out *in addition* to the General Checklist:
- [ ] I have tested the sample on at least one compliant Vulkan implementation
- [ ] If the sample is vendor-specific, I have [tagged it appropriately](https://github.com/KhronosGroup/Vulkan-Samples/tree/main/CONTRIBUTING.adoc#General-Requirements)
- [ ] I have stated on what implementation the sample has been tested so that others can test on different implementations and platforms
- [ ] Any dependent assets have been merged and published in downstream modules
- [ ] For new samples, I have added a paragraph with a summary to the appropriate chapter in the readme of the folder that the sample belongs to [e.g. api samples readme](https://github.com/KhronosGroup/Vulkan-Samples/blob/main/samples/api/README.adoc)
- [ ] For new samples, I have added a tutorial README.md file to guide users through what they need to know to implement code using this feature. For example, see [conditional_rendering](https://github.com/KhronosGroup/Vulkan-Samples/tree/main/samples/extensions/conditional_rendering)
- [ ] For new samples, I have added a link to the [Antora navigation](https://github.com/KhronosGroup/Vulkan-Samples/blob/main/antora/modules/ROOT/nav.adoc) so that the sample will be listed at the Vulkan documentation site

### CODE REVIEW

File: docs/build.adoc [hunk 1]
```diff
@@ -260,7 +260,7 @@ cmake --build build/mac --config Release --target vulkan_samples -j4
 `Step 3.` Run the *Vulkan Samples* application to display the help message
 
 ----
-./build/mac/app/bin/Release/x86_64/vulkan_samples --help
+./build/mac/app/bin/Release/arm64/vulkan_samples --help
```
Comment (chmosquera) [inline docs/build.adoc]:
```suggestion
./build/mac/app/bin/Release/arm64/vulkan_samples.app/Contents/MacOS/vulkan_samples --help
```

**Comment by SRSaunders** [comment]
See item 7 in pending pull request #1084.  I am proposing to revert to a macOS command line app vs. an app bundle since it is not required for this project (other than for iOS).  In addition, this PR also contains a change to the docs (https://github.com/KhronosGroup/Vulkan-Samples/pull/1084/commits/53212364e2acf98d9a5732b84533325d8890b2af) for <arm64|x86_64> path options.

**Comment by chmosquera** [comment]
Thanks for letting me know! In that case, we can probably close this PR. It seems like with the two PRs mentioned above, this change won't be needed.

**Comment by SRSaunders** [comment]
Thanks for reviewing and looking into this.

**Pull request closed without merging**